.github/workflows/build.yml

```diff
@@ -345,19 +345,59 @@ jobs:
       - id: printtag
         name: Output image name and tag
         run: echo "IMAGE_TAG=$(make -e print-docker-tag)" >> $GITHUB_OUTPUT
-  openshift_preflight:
-    name: Run the OpenShift Preflight check on the published images
-    needs:
-      - package_and_publish
-    runs-on: ubuntu-latest
-    env:
-      IMAGE_TAG: ${{ needs.package_and_publish.outputs.IMAGE_TAG }}
+
+  build_stackablectl:
```

Comment:
I guess we should only build this when we actually release a new version (pushing a tag). This however should be tackled in a separate PR, which I can push forward.

Reply:
The idea is that we build for all platforms during our normal development process, so that we get notified as quick as possible when the build for a certain platform breaks.
Agreed that we need this on release, happy to have that in a separate PR

Reply:
Okay then let's include the release process in a separate PR. I will create one and work on this the next few days.

Reply:
Awesome!